parts/common/components.json

```diff
@@ -6,23 +6,28 @@
       "multiArchVersionsV2": [],
       "windowsVersions": [
         {
+          "comment": "should have latest version and natest version -1",
```

Comment:
typo: natest

Reply:
Annoyingly github auto-merged the PR even tho you'd written a comment! I don't like that behaviour. Will fix this in a future PR.